common/common.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * This file and its contents are supplied under the terms of the
+ * Common Development and Distribution License ("CDDL"), version 1.0.
+ * You may only use this file in accordance with the terms of version
+ * 1.0 of the CDDL.
+ *
+ * A full copy of the text of the CDDL should have accompanied this
+ * source.  A copy of the CDDL is also available via the Internet at
+ * http://www.illumos.org/license/CDDL.
+ */
+
+/*
+ * Copyright 2019, Joyent, Inc.
+ */
+
+#ifndef _COMMON_H
+#define	_COMMON_H
+
+#include <bunyan.h>
+#include <synch.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef GUID_LEN
```

Comment:
It was originally to avoid having to include "pivy/piv.h" everywhere which also defines it, but it seems like doing so isn't causing any harm (or forcing the stuff other than kbmd to link in more stuff than is necessary), so I'll just include "pivy/piv.h" (with a comment indicating why it's being included).

Reply:
confirmed.